app/models/form/sales/questions/about_price_value_check.rb

```diff
@@ -2,8 +2,7 @@ class Form::Sales::Questions::AboutPriceValueCheck < ::Form::Question
   def initialize(id, hsh, page)
     super
     @id = "value_value_check"
-    @check_answer_label = "Purchase price confirmation"
-    @header = "Are you sure?"
+    @copy_key = "sales.property.value_value_check"
```

Comment:
I couldn’t find another section where this question is used. It seems to be relevant only to the property information.

Reply:
It references value, so I assume it appears in the sale information section (for each sale type) as well?

Reply:
Ah yes it does appear in some other subsections (outright_sale, discounted_ownership_scheme and shared_ownership_scheme). I'll remove this translation.

Reply:
I think they want to be in a separate file rather than removed - the relevant text on soft validation questions/pages do want to be editable in the same way as other things, so we want to pull them into translations, just structurally it doesn't make sense for them to be inside section files.

Reply:
Within this branch?

Reply:
Yes - This ticket includes soft validation questions used in this section (in large part so that we should then have examples of everything for overall structure, making future tickets much simpler).